src/styles/UserPage.css

```diff
@@ -0,0 +1,44 @@
+.user-main-container {
+	height: 100%;
+}
+
+.user-page-container {
+	margin-left: 1rem;
+	border-radius: 1rem;
+}
+
+.flex-align-center {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+}
+.user-form-input {
+	padding: 0.5rem;
+}
+
+.pad-normal-top {
+	padding-top: 1rem;
+}
+
+.padding-large-top {
+	padding-top: 2rem;
+}
+
+.padding-normal-top {
+	padding-top: 1rem;
+}
```

Comment:
.pad-normal-tap and .padding-normal-top could be combined since they have the same property

Reply:
Okay, I'll Do that, thank you Sunil